File: leetcode-submissions/1945_Sum_of_Digits_of_String_After_Convert.ts
function getLucky(s: string, k: number): number {
    let number:string="";
    for(const ele of s){
        let ch=ele.charCodeAt(0)-97;
        number+=`${ch+1}`;

    }
    while(k--){
        let num=0;
        for(const ele of number){
            num+=parseInt(ele);
        }
        number=num.toString();
    }
    return parseInt(number);
};

File: leetcode-submissions/1945_Sum_of_Digits_of_String_After_Convert.cpp
class Solution {
public:
    int getLucky(string s, int k) {
        string number="";

        for(int i=0;i<s.size();i++){
            int ch=s[i]-97;
            number+=to_string(ch+1);
            
        }
       int ans=0;
        while(k--){
            int num=0;
        for(int i=0;i<number.size();i++){
            num=num+(number[i]-'0');
        }
        number=to_string(num);

        }
        return stoi(number);
    }
};